editoast/src/schema/mod.rs

```diff
@@ -404,6 +404,7 @@ pub struct Sign {
     pub angle_geo: f64,
     pub angle_sch: f64,
     pub side: Side,
+    pub direction: ApplicableDirections,
```

Comment:
I agree with the idea of adding a direction information. This is a modification of the railjson format that implies to:
- Adapt python libraries
- Adapt import
- Make a migration
- Bump railjson version
- ~Adapt core~

We can remove `angle_geo` and `angle_sch`. Also we should use `Direction` and not `ApplicableDirections`.

```suggestion
    pub side: Side,
    pub direction: Direction,
```